doc/languages-frameworks/python.section.md

```diff
@@ -1384,7 +1391,7 @@ If you want to create a Python environment for development, then the recommended
 method is to use `nix-shell`, either with or without the `python.buildEnv`
 function.
 
-### How to consume Python modules using pip in a virtual environment like I am used to on other Operating Systems?
+### How to consume Python modules using pip in a virtual environment like I am used to on other Operating Systems? {#how-to-consume-python-modules-using-pip-in-a-virtual-environment-like-i-am-used-to-on-other-operating-systems}
```

Comment:
Man this anchor is kind of ridiculous.

Reply:
Yeah, initially I wanted to comb up the anchors but in the end I decided against it as it was too much work, the PR was already too big and it might break someone’s link.